### ISSUE TITLE
Relationship between sketchbooks parent records and their pages

### ISSUE THREAD

**Issue by edgartdata** [ycba-cia/blacklight-collections2#197]
As for issue #82 I am adding an entry to discuss mechanisms across collections and maybe types of objects to relate parent and child records, such as with sketchbooks. 

@yulgit1 Unlike issue #82, and because there can be many pages to one object, I don't think it will be possible to have a static list the way we currently do for frames and paintings. Happy to brainstorm about this in a few weeks after June 24 :)

**Comment by flapka** [comment]
@edgartdata @yulgit1 Yes, this could be useful across collection. Obvious utility for archival descriptions, when those are integrated. 

There are less numerous examples in MARC (probably fewer than 100 or 200, altogether). On the other hand, RB has numerous "bound with" objects. These relationships are those of a sibling nature, but it could still be nice to have a better linking mechanism than we currently have.

Most library catalog interfaces don't handle the "bound with" relationship well, but here's an example of one that does (scroll down to the Holdings section): https://iucat.iu.edu/catalog/18375799

**Comment by edgartdata** [comment]
An example of how hierarchy is visualized in archives: http://findit.library.yale.edu/catalog/digcoll:554296

**Comment by edgartdata** [comment]
https://digitalcollections.nypl.org/items/8776c530-0e52-0134-d208-00505686a51c

**Comment by edgartdata** [comment]
@edgartdata to look into passing TMS parent child relationships to LIDO (for frames-paintings and other complex objects).
On the library side, there is work underway to express such complex objects.
